Taskfile.yml

```
@@ -44,12 +46,31 @@ tasks:
       - task compose -- run --rm node yarn --cwd /app/web/themes/custom/itkdev/itkdev_base_theme watch
 
   simulate-github-actions:
+    desc: "Simulate most github actions to prepare the code and catch errors before github does"
     cmds:
       - task composer -- normalize
       - task check-code
 
   check-code:
+    desc: "Check coding standards"
     cmds:
       - docker run --rm --volume "$PWD:/md" peterdavehello/markdownlint markdownlint $(git ls-files *.md)
       - task composer -- code-analysis
       - task composer -- coding-standards-check/twig-cs-fixer
+
+  apply-fixtures:
+    desc: "Apply fixtures"
+    cmds:
+      - task drush -- pm:install ai_screening_fixtures_base
+      - task drush -- content-fixtures:load -y
+      - task drush -- pm:uninstall ai_screening_fixtures_base content_fixtures
+
+  apply-fixtures-groups:
+    desc: "Apply one or more groups of fixtures"
+    cmds:
+      - task drush -- pm:install ai_screening_fixtures_base
+      - task drush -- content-fixtures:load --groups={{.FIXTURE_GROUPS}} -y
+      - task drush -- pm:uninstall ai_screening_fixtures_base content_fixtures
+    requires:
+      vars:
+        - FIXTURE_GROUPS
```

Rather than creating a new task for handling groups, you can use `{{.CLI_ARGS}}` in the task commands: 

```suggestion
  apply-fixtures:
    desc: "Apply fixtures"
    cmds:
      - task drush -- pm:install ai_screening_fixtures_base
      - task drush -- content-fixtures:load --yes {{.CLI_ARGS}}
      - task drush -- pm:uninstall ai_screening_fixtures_base content_fixtures
```

and then call it like

``` shell
task apply-fixtures -- --groups=base,user
```

(see https://unix.stackexchange.com/a/11382, say, for details on what ` -- ` means).

This way we can pass _any_ options and flags to the `content-fixtures:load` Drush command.